MobileCenter/MobileCenter/Internals/Application Delegate/MSAppDelegateForwarder.m

```diff
@@ -286,19 +287,28 @@ - (void)custom_application:(UIApplication *)app
     didReceiveRemoteNotification:(NSDictionary *)userInfo
           fetchCompletionHandler:(void (^)(UIBackgroundFetchResult))completionHandler {
 
+  // Collect `UIBackgroundFetchResult` from delegates in order to call the original completion handler later.
+  __block UIBackgroundFetchResult forwardedFetchResult = UIBackgroundFetchResultNoData;
+  void (^forwardedCompletionHandler)(UIBackgroundFetchResult) = ^(UIBackgroundFetchResult fetchResult) {
+    forwardedFetchResult = fetchResult;
```

Comment:
Yes exactly, that's why I added the FIXME just below :). Right now there only one delegate implementing this selector so it's fine but we will have to fix it yes.